wagtail/wagtailadmin/views/account.py

```diff
@@ -141,3 +141,16 @@ def logout(request):
     request.session.modified = False
 
     return response
+
+
+def change_avatar(request):
+    if request.method == 'POST':
+        form = AvatarPreferencesForm(request.POST, request.FILES, instance=UserProfile.get_for_user(request.user))
+        if form.is_valid():
+            form.save()
+            messages.success(request, _("Your preferences have been updated successfully!"))
+            return redirect('wagtailadmin_account_change_avatar')
+    else:
+        form = AvatarPreferencesForm(instance=UserProfile.get_for_user(request.user))
+
+    return render(request, 'wagtailadmin/account/change_avatar.html', {'form': form})
```

Comment:
I've used that in order to be consistent with other parts of the code. Exactly: https://github.com/wagtail/wagtail/blob/master/wagtail/wagtailadmin/views/account.py#L89 , I'd prefer keeping as this for now. Is this ok?